keps/168-pending-workloads-visibility/README.md

```diff
@@ -530,9 +530,18 @@ Third iteration (0.7):
 
 - reevaluate the need for exposing positions and support if needed
 
-#### Stable
+#### Deprecation
 
-- drop the feature gate
+We deprecate the feature, and going to remove it in feature versions of the API,
+in favor of monitoring pending workloads using the [on-demand visibility API](https://kueue.sigs.k8s.io/docs/tasks/manage/monitor_pending_workloads/pending_workloads_on_demand/).
+
+First iteration (0.9):
+
+- deprecate the QueueVisibility feature gate and corresponding API.
+
+Second iteration (v1beta2):
+
+- remove the QueueVisibility feature gate and corresponding API.
```

Comment:
I think that this is no available since once we remove the corresponding APIs, the backward compatibility will be lost whether or not we use the QueueVusibilitiy. So, we need to bump the API version instead of removing the field.

Reply:
I think this is implied by the line above saying `v1beta2`, but we might be more explicit. Say:
"remove the QueueVisibility feature gate and corresponding API on bumping the API version."

Reply:
SGTM, could you mention, which API object should be bumped to v1beta2?
I guess that Config and ClusterQueue, right?

Reply:
I think so

Reply:
LocalQueue also.

Reply:
for CQ we have [this](https://github.com/kubernetes-sigs/kueue/blob/9d1efed4ffac5ed20825a11dc8760c485b52b2e0/apis/kueue/v1beta1/clusterqueue_types.go#L290-L293), I didn't see any API exposed for LQ, but feel free to add it if I'm missing something.

Reply:
> LocalQueue also.

What is the reason?

Reply:
Ah, ok. I thought we need to deprecate [this](https://github.com/kubernetes-sigs/kueue/blob/9d1efed4ffac5ed20825a11dc8760c485b52b2e0/apis/kueue/v1beta1/clusterqueue_types.go#L266-L269) too.

Reply:
Done